grant/grant.go

```diff
@@ -307,6 +307,20 @@ type RedeemGrantsRequest struct {
 	Transaction string      `json:"transaction" valid:"base64"`
 }
 
+// DecodeGrants decodes grants and sends back a new array
+func (req *RedeemGrantsRequest) DecodeGrants() ([]Grant, error) {
```

Comment:
I think it's more natural to make this:
```
func DecodeGrants([]string) ([]Grant, error) {
```
so it isn't tied to a particular request type